client/components/info-popover/index.jsx

```diff
@@ -11,84 +11,85 @@ import Popover from 'components/popover';
 import classNames from 'classnames';
 import analytics from 'lib/analytics';
 
-export default React.createClass( {
+export default class InfoPopover extends Component {
+	static propTypes = {
+		autoRtl: PropTypes.bool,
+		className: PropTypes.string,
+		gaEventCategory: PropTypes.string,
+		id: PropTypes.string,
+		ignoreContext: PropTypes.shape( {
+			getDOMNode: PropTypes.func
+		} ),
+		popoverName: PropTypes.string,
+		position: PropTypes.oneOf( [
+			'top',
+			'top right',
+			'right',
+			'bottom right',
+			'bottom',
+			'bottom left',
+			'left',
+			'top left',
+		] ),
+		rootClassName: PropTypes.string,
+	};
+
+	static defaultProps = {
+		autoRtl: true,
+		position: 'bottom',
+	};
 
-	displayName: 'InfoPopover',
+	state = { showPopover: false };
 
-	propTypes: {
-		id: React.PropTypes.string,
-		position: React.PropTypes.string,
-		className: React.PropTypes.string,
-		rootClassName: React.PropTypes.string,
-		gaEventCategory: React.PropTypes.string,
-		popoverName: React.PropTypes.string,
-		ignoreContext: React.PropTypes.shape( {
-			getDOMNode: React.PropTypes.function
-		} ),
-	},
+	handleClick = ( event ) => {
+		event.preventDefault();
+		this.setState( {
+			showPopover: ! this.state.showPopover },
```

Comment:
Indentation/formatting seems broken here, can we fix it? I'd expect something like:

```
this.setState( {
	showPopover: ! this.state.showPopover
}, this.recordStats );
```